manifests/controller-manager/kubeadm.conf

```diff
@@ -1,12 +1,20 @@
-kind: MasterConfiguration
-apiVersion: kubeadm.k8s.io/v1alpha2
-kubernetesVersion: 1.12.3
-unifiedControlPlaneImage: "gcr.io/google_containers/hyperkube-amd64:v1.12.3"
-cloudProvider: external
-apiServerExtraArgs:
-  enable-admission-plugins: NodeRestriction,Initializers
-  runtime-config: admissionregistration.k8s.io/v1alpha1
-controllerManagerExtraArgs:
-  external-cloud-volume-plugin: openstack
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: InitConfiguration
+nodeRegistration:
+  kubeletExtraArgs:
+    cloud-provider: "external"
+---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+kubernetesVersion: stable
 networking:
-  podSubnet: 192.168.0.0/24
+  podSubnet: 10.244.0.0/16
+controllerManager:
+  extraArgs:
+    "external-cloud-volume-plugin": "openstack"
```

Comment:
if using CSI, then as i know there is no need for  `external-cloud-volume-plugin`

Reply:
I will double-check here. This was in the old documentation, however, my previous troubles in testing may have been unrelated to this.

Reply:
Tested and confirmed not required. Removed from the commit.